gen/generate.go

```diff
@@ -79,7 +90,26 @@ func GenerateGenesisFiles(genesisData GenesisData, consensus config.ConsensusPro
 	}
 
 	if sum != TotalMoney {
-		panic(fmt.Sprintf("Amounts don't add up to TotalMoney - off by %v", int64(TotalMoney)-int64(sum)))
+		fsum := float64(sum)
```

Comment:
I'm not sure if this is the best place to do this adjustment, but the idea is good.

Reply:
GenesisData.Wallets[].Stake is a float64 [0..100.0], this is where we convert it to uint64 microalgos, so I think it fits in this function.

Reply:
sorry - my bad. I missed the `Stake:  uint64(float64(TotalMoney/100)*wallet.Stake + .5)` above, and thought it was implemented elsewhere.